src/main/riddl/Organization/organizationTypeDefinitions.riddl

```diff
@@ -1,4 +1,4 @@
-type Status is any of {Draft, Active, Suspended, Terminated} briefly "Internal status of an Organization."
+type StatusUpdates is any of {Active, Suspended} briefly "Internal status of an Organization."
```

Comment:
Inactive is different from Suspended for Members in that Suspended is an explicit action to be taken by an admin, while Inactive is a state the system uses when a Member has not been using the app for a while. Not sure if we want to measure the same for Orgs, but I don't see why not really.

Reply:
It seems that "Inactive", per your definition,  could be a state that is modeled in Akka as "passivated" but I wouldn't want to make a state change just to passivate an actor. To disambiguate, we need to understand the behavior differences are in the "Inactive" state. If Member doesn't behave any differently between Active or Inactive then Inactive isn't useful.  If that's the case, then remove Inacive from Members and don't add it to any entity definition. The only reason to have a State Machine state is to capture either data or behavior differences.  If neither changes, the state isn't needed.

Reply:
It's important to keep in mind that there is a difference between a state that uses its own handlers and a state that is enumerated and so shares handlers with other states. Hence the desire to disambiguate using `Status`es for the latter.

Inactive and Suspended and both cases of the latter, so we will not be changing states in a heavy way, only by noticing that the enum has a certain value and thus hitting a point in the handler flow that passivates the state.

Reply:
Agreed, but did you get my point above?

I don't think "Inactive" is a valid state. Unless you can show a case where "Inactive" has a handler that is definitionally different than that of the other states, it should just be removed, everywhere.  I'm basing this on your assertion that "Inactive is a state the system uses when a Member has not been using the app for a while".  In other words, this is time-out transition the system takes in that entity on its own. There is no command needed externally to force the transition to Inactive.  This happens during passivation of actors.  We don't need this state in our model because it has nothing to do with the business logic .... UNLESS you can show how the Inactive entity responds to messages differently than Active ones.  If the Active and Inactive states are identical then one of them is redundant and I'm suggesting removal of "Inactive".

Reply:
That does make sense. There is no different business logic, the difference in state is entirely technical. I hadn't thought about it that way.